### ISSUE TITLE
plz support Luna Unit Frames  thx！！

### ISSUE THREAD

**Issue by sylvanas1408** [jordonwow/bigdebuffs#45]
Can you please add support to Luna Unit Frames? It's extremely similar to Shadowed Unit Frames.
thx very much

**Comment by sylvanas1408** [comment]
@jordonwow

**Comment by VanDeAnna** [comment]
yep. LunaUnitFrame is a lightweight and efficient unit frame addon.
It would be appreciated if it one day could support LUF.

Thanks for your great work.
